services/ui-src/src/forms/mcpar/index.ts

```diff
@@ -18,6 +18,12 @@ export const mcparReportJson = {
   validationJson: compileValidationJsonFromRoutes(mcparReportRoutesFlat),
 };
 
-export const nonFormPages = ["/mcpar/get-started"];
-export const isMcparReportFormPage = (pathname: string): boolean =>
-  pathname.includes("/mcpar/") && !nonFormPages.includes(pathname);
+export const isMcparReportFormPage = (pathname: string): boolean => {
+  let i = 0;
+  for (i; i < mcparReportRoutesFlat.length; i++) {
+    if (mcparReportRoutesFlat[i].path === pathname) {
+      return true;
+    }
+  }
+  return false;
+};
```

Comment:
```suggestion
export const isMcparReportFormPage = (pathname: string): boolean => {
  return mcparReportRoutesFlat.map((route) => route.path).includes(pathname);
};
```

Reply:
Alternatively, for short-circuiting behavior:
```ts
  return mcparReportRoutesFlat.some((route) => route.path == pathname);
```

Reply:
For sure! Since this code is pretty self explanatory the benefit or .map/.some/.forEach's readability against the speed was something I felt speed should win out on. That being said I'm happy to swap it out if you feel that it would be better to use this method.

Reply:
For comparison, heres a quick performance test I wrote:

![Screenshot 2023-04-17 at 11 46 46 AM](https://user-images.githubusercontent.com/19439679/232540113-af025b54-8e1c-4ea9-b285-fbfd7ee9de72.png)

Reply:
If I'm reading this right, the `array.some` approach would be able to execute 1900 times per second, even if we had 100,000 routes? That's absolutely fast enough that I would rather take the performance penalty than have an index-iterating for loop - especially given that our route count is more likely to be in the 10s than the 100,000s.